test/controllers/vat/ClaimRefundControllerSpec.scala

```diff
@@ -1,4 +1,20 @@
-package controllers.sa
```

Comment:
I don't think this should be here. generated-test directories are removed after running migration.  Do we need it?

Reply:
couldnot find this.